kratos/python_scripts/assign_initial_state_process.py

```diff
@@ -0,0 +1,125 @@
+# Importing the Kratos Library
+import KratosMultiphysics
+
+
+def Factory(settings, Model):
+    if not isinstance(settings, KratosMultiphysics.Parameters):
+        raise Exception(
+            "expected input shall be a Parameters object, encapsulating a json string"
+        )
+    return SetInitialStateProcess(Model, settings["Parameters"])
+
+
+# All the processes python processes should be derived from "Process"
+
+
+def components_to_string(vect):
+    v = []
+    for c in vect:
+        if c.IsNumber():
+            s = str(c.GetDouble())
+        elif c.IsString():
+            s = c.GetString()
+        else:
+            msg = "SetInitialStateProcess: Vector component must be scalar or string"
+            raise Exception(msg)
+        function = KratosMultiphysics.PythonGenericFunctionUtility(s)
+        if function.DependsOnSpace():
+            msg = "SetInitialStateProcess: 'x', 'y' and 'z' variables not supported yet"
+            raise Exception(msg)
+        v.append(function)
+    return v
+
+
+class SetInitialStateProcess(KratosMultiphysics.Process):
```

Comment:
test is missing

Reply:
Indeed

Reply:
@marandra can you do this? Otherwise I can find some time to do that

Reply:
sure